contrib/oc-environment.sh

```diff
@@ -12,6 +12,7 @@
 # The environment variables beginning with "BRIDGE_" act just like bridge
 # command line arguments - in fact. to get more information about any of them,
 # you can run ./bin/bridge --help
+set -x
```

Comment:
Is it necessary to print all executed commands into terminal ?

Reply:
Opened https://github.com/openshift/console/pull/6040 to remove it

Reply:
Oops my bad... Thanks for fixing it Andy!